templates/blog/macros.j2

```diff
@@ -4,24 +4,26 @@
 
 
 {% macro list(post, section='blog-post') %}
-  <article {{ utility.attr_if('data-module', section) }}>
+  <article {{ utility.attr_if('data-module', section) }} class="preview">
     {% set heading = 'h2' if section else 'h3' %}
-    <{{ heading }}>
+    <{{ heading }} class="post-title">
       <a href="{{ link_to('page', slug=post.slug) }}" rel="bookmark">
         {{ post.title }}
       </a>
     </{{ heading }}>
 
-    <p>
-      {{ utility.datetime(post.pub_date) }}
+    <div class="meta">
+      <p class="byline">
+        <span class="meta-bit">by</span> {{ birds.get_card(post.author) }}
+        <span class="meta-bit">on</span> {{ utility.datetime(post.pub_date) }}
+      </p>
       {{ blog.tags(post.tags) }}
-    </p>
+    </div>
 
-    {{ birds.get_card(post.author, details=true) }}
     {{ blog.header_image(post) }}
 
     {% if post.summary %}
-      <p>
+      <p class="summary">
```

Comment:
This doesn't work. I know we were going to look into making it work, but I think it got dropped. Was it @jgerigmeyer and I that had that discussion?